lib/tram/policy/errors.rb

```diff
@@ -0,0 +1,44 @@
+require "tram/policy/error"
+
+module Tram
+  class Policy
+    class Errors
+      include Enumerable
+      extend Forwardable
+
+      def_delegators :@errors, :each, :size, :empty?
+      attr_reader :errors
+
+      def initialize(policy)
+        @policy = policy
+        @errors = []
```

Comment:
It's better to use `set` here -- for two reasons (1) errors aren't ordered, (2) they should be unique

PS. That's why we needed `Error#==`